timeline_logger/manager.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Manager, Model
+
+
+class TimelineLogManager(Manager):
+    def for_object(self, obj: Model):
+        assert isinstance(obj, Model)
+
+        app_label = obj._meta.app_label
+        model_name = obj._meta.model_name
```

Comment:
```suggestion
```

we already have the type annotation to say it's a model, that should be sufficient

The `obj._meta` stuff isn't needed because we can use `ContentType.objects.get_for_model(...)` which does this under the hood.